public/scripts/worktrollyUI.js

```diff
@@ -67,4 +86,22 @@ $("#submitCreateNewTask").click(function() {
         newPage = "bodyContent";
         uiLoader();
     });
+});
+
+$("#submitNewSprint").click(function() {
+    var startDate = $("#startdateNewSprint").val();
+    var endDate = $("#enddateNewSprint").val();
+    var status = $("#statusNewSprint").val();
+
+
+    console.log(startDate);
+    console.log(endDate);
+    console.log(status);
+
+    var startNewSprintFunction = firebase.functions().httpsCallable('startNewSprint');
+    startNewSprintFunction({ Startdate: startDate, Enddate: endDate, Status: status }).then(result => {
```

Comment:
Fix the Naming convention of Startdate, Enddate.

Reply:
Changed naming convention